docs/google/engine.md

```diff
@@ -139,6 +139,16 @@ number of repetitions.
 Batching circuits together that do not follow these restrictions may not
 cause an error, but your performance will not be significantly improved.
 
+Results can be retrieved in two different forms.  If the resulting
+`EngineJob.results()` is called, the results will be returned in a
```

Comment:
Consider specifying the two techniques in a list for easier parsing of the separate methods.

Reply:
Good idea.  Done.